scripts/hello-world-deploy.sh

```diff
@@ -28,4 +28,5 @@ fi
 
 npm install
 
-node hello-world-app.js
\ No newline at end of file
+node hello-world-app.js&
```

Comment:
You should try using the package.json commands to start:

`node dev start` or `node start-hello` like in your package.json